install.sh

```diff
@@ -963,6 +963,7 @@ create_env_file() {
     $SUDO chmod 0600 ${FILE_K3S_ENV}
     sh -c export | while read x v; do echo $v; done | grep -E '^(K3S|CONTAINERD)_' | $SUDO tee ${FILE_K3S_ENV} >/dev/null
     sh -c export | while read x v; do echo $v; done | grep -Ei '^(NO|HTTP|HTTPS)_PROXY' | $SUDO tee -a ${FILE_K3S_ENV} >/dev/null
+    sh -c export | while read x v; do echo $v; done | grep -E 'PATH' |  sed "s/'//g" | $SUDO tee -a ${FILE_K3S_ENV} >/dev/null
```

Comment:
Why are we adding the user’s PATH to the systemd unit’s PATH? Generally system services have a path that is intentionally different from user paths, it is not really a best practice to add random user paths to the service. If users have non-standard paths I think they should have to add these manually, I would not like to just add them unconditionally.